### PR TITLE
implement best practices for mails (WIP)

### LINKED ISSUE #616: improve welcome E-Mail
- Die Mail hat aktuell keine Zeilenabstände
- Der Link ist nicht als Link formaitert

**Des Weiteren würde ich sie so umformulieren:**

Hallo SimonTestaccount,

wir freuen uns, dass du einen Kommentar schreiben oder an serlo.org mitarbeiten möchtest!

Wo und wie du dich beteiligen kannst, erfährst du auf der Community-Startseite: https://de.serlo.org/community

Bei Fragen oder Problemen, schreib uns einfach an de@serlo.org.

Viel Spaß mit Serlo!

### LINKED ISSUE #661: implement spam guidelines for emails
e.g. unsubscribe link in notifications emails

### LINKED ISSUE #839: idea to improve notification
differentiate between content notifications and comment notifications.

Hallo Simon!

(mehrere notifications) User auf serlo.org haben auf Kommentare von dir reagiert:
(eine notification) Ein user auf serlo.org hat auf einen Kommentar von dir reagiert:

- ...
- ...
 
(mehrere notifications) Hier ist eine Übersicht, was mit Inhalten passiert ist, die du erstellt oder bearbeitet hast:
(eine notification) Das ist mit einem Inhalt passiert, den du erstellt oder bearbeitet hast:

- ...
- ...

 
Zu welchen Inhalten du notifications erhältst, kannst du in deinen [Einstellungen ändern]. Alle weiteren Informationen findest du auf der [Community-Startseite].

Dein Serlo-Team

### PR DESCRIPTION
**Pull request by Knorrke**
- [X] seperate Renderer from Listeners and include plain text
- [X] improve texts

TODO: 
- [ ] Check if this works, also unittest!
- [x] rewrite `.phtml` files to `.twig`

closes #661, closes #616, closes #839

### CODE REVIEW

File: src/module/Markdown/src/Markdown/Service/OryRenderService.php [hunk 1]
```diff
@@ -50,9 +50,9 @@ public function render($input)
     {
         $key = 'editor-renderer-' . hash('sha512', $input);
 
-        if ($this->storage->hasItem($key)) {
-            return $this->storage->getItem($key);
-        }
+//        if ($this->storage->hasItem($key)) {
```
[the remark this answers is not in the document]
Comment (Knorrke) [reply src/module/Markdown/src/Markdown/Service/OryRenderService.php]: Oh^^ thanks, didn't notice that I commited that.

**Comment by Knorrke** [comment]
Now also closes #839
TODO: add plain text version for notification mail
- [ ]

**Comment by inyono** [comment]
Well, this was annoying... 😬 But I got the service locator to work in PHPUnit, see https://github.com/serlo-org/athene2/tree/feature/tests-service-locator. It did catch a problem with the current template (see `FIXME` comment) and might need some changes to get the other templates to work (we can pair up for that later today if you want).

**Note**: the tests should now run in the docker container (e.g. since you need the `gettext` php extension).

- [ ] Check Travis

**Comment by Knorrke** [comment]
superseded by #857

**Pull request closed without merging**